group/assignment5/src/Graph.java

```diff
@@ -45,103 +50,101 @@ public void addEdge(T parent, T child) {
     /**
      * This method adds a new vertex in a directed graph.
      *
-     * <p>Graph vertices are unique - only one vertex can be created for each T value
+     * Graph vertices are unique - only one vertex can be created for each T value
      * @param value is a value of a new vertex
      */
     public void addVertex(T value) {
         vertexByValue.put(value, new Vertex(value));
     }
 
-    /**
-     * This method is to find one of possible topological
-     * orders (if a graph has more than one) of a graph vertices
-     * @return topological ordered list of vertices of a graph
-     */
-    public List<T> getTopologicalOrder() {
-        List<T> alphabet = new ArrayList<>();
-        dfs(alphabet);
-        Collections.reverse(alphabet);
-        vertexByValue.values().forEach(e -> e.setState(VertexState.UNVISITED));
-        return alphabet;
-    }
 
     /**
      * This method is to find all possible orders of topological
      * of vertices of the graph
      * @return all topological orders of vertices of a graph
      */
     public List<List<T>> getAllTopologicalOrders() {
-        List<List<T>> alphabets = new ArrayList<>();
-        List<T> alphabet = new ArrayList<>();
-        dfsForAllTopologicalOrders(alphabet, alphabets);
-        vertexByValue.values().forEach(e -> e.setState(VertexState.UNVISITED));
-        return alphabets;
+        List<List<T>> allOrderedVerticesVal = new ArrayList<>();
```

Comment:
I've moved each dfs function next to the caller function so the class will be more readable.